LICENSE

```diff
@@ -1,6 +1,5 @@
-Flynn® is a trademark of Prime Directive, Inc.
```

Comment:
Will Licensee still detect properly if we leave this line in?

Reply:
Unfortunately, no - Licensee will not detect properly if this line is kept. It would instead output:
```
License:        Other
Matched files:  LICENSE
LICENSE:
  Content hash:  c732b26b776de36f1c95653afc855f9051fe15c9
  License:       Other
  Closest licenses:
    BSD-3-Clause similarity:        95.08%
    BSD-3-Clause-Clear similarity:  92.49%
    BSD-2-Clause similarity:        87.72%
```